### ISSUE TITLE
[Bug]: Inconsistent spacing in mobile view

### ISSUE THREAD

**Issue by rohinish404** [FrancescoXX/free-Web3-resources#404]
### Description

There is inconsistent spacing in the section shown in the below image.
<img width="495" alt="Screenshot 2023-02-22 at 11 52 55 PM" src="https://user-images.githubusercontent.com/92542124/220721404-3cc32f14-2c34-45a6-89df-c00d8460d93c.png">

### To Reproduce

1. Go to homepage
2. Scroll down to "easy to use"
3. See error

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

**Comment by tobySolutions** [comment]
Fixed by #403

**Comment by rohinish404** [comment]
i think you closed the wrong issue..the logo issue should be closed becoz it completed.

**Comment by tobySolutions** [comment]
Thank you @rohinish404!!

**Comment by rohinish404** [comment]
np @tobySolutions, i'll make a pr soon!

**Comment by tobySolutions** [comment]
Awesome! I'll be expecting.

**Comment by tobySolutions** [comment]
Resolved by #406
